vpr/src/base/partition_region.h

```diff
@@ -46,7 +46,7 @@ class PartitionRegion {
      *   @param pr1     One of the PartitionRegions to be intersected
      *   @param pr2     One of the PartitionRegions to be intersected
      */
-    friend PartitionRegion intersection(PartitionRegion& pr1, PartitionRegion& pr2);
+    friend PartitionRegion intersection(const PartitionRegion& cluster_pr, const PartitionRegion& new_pr);
 
```

Comment:
add declaration of void version of this function

Reply:
also change the name and the first parameter would not be const (could also have it be a bool that returns whether or not there is an intersect)